sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1496,6 +1514,8 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   rec.ncrt_hits       = srcrthits.size();
   rec.crt_tracks        = srcrttracks;
   rec.ncrt_tracks       = srcrttracks.size();
+  rec.opflashes       = srflashes;
```

Comment:
Again, pre-existing pattern, but why not fill straight into `rec.opflashes` above?

Reply:
Exactly as you say, pre-existing pattern. I just was not sure if I should go against the apparent grain, but I can change it.